src/modules/settings.js

```diff
@@ -0,0 +1,17 @@
+export const SYSTEM_SETTINGS = [
+    //'keyDateFormat',
+    //'keyAnalysisRelativePeriod',
+    //'keyAnalysisDigitGroupSeparator',
+    //'keyIgnoreAnalyticsApprovalYearThreshold',
+    'keyGatherAnalyticalObjectStatisticsInDashboardViews',
+];
+
+export const extractUserSettings = settings => {
+    const nameProp = settings.keyAnalysisDisplayProperty;
```

Comment:
This function does not seem to be used for anything.

Reply:
Agreed. Left over from importing from data-visualizer-app. Removing along with rest of modules/settings.